src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { formatDateShort, formatDateLong } from '../utils/helpers';
+
+class EnrollmentHistory extends React.Component {
+  render() {
+    const enrollmentData = this.props.enrollmentData;
+    const today = formatDateShort(new Date());
+
+    // TODO: find out when this warning should be shown.
+    let expirationWarning;
+    if (true) { // eslint-disable-line no-constant-condition
+      expirationWarning = (
+        <div className="usa-alert usa-alert-warning usa-content">
+          <div className="usa-alert-body">
+            <h2>Your benefit expires on {formatDateLong(enrollmentData.delimitingDate)}</h2>
+            You have {enrollmentData.remainingEntitlement} months left on your Chapter 33 Post-9/11 GI Bill benefit before it expires on {formatDateShort(enrollmentData.delimitingDate)}.
+          </div>
+        </div>
+      );
+    }
+
+    // TODO: find out when this warning should be shown.
+    let trainingWarning;
+    if (true) { // eslint-disable-line no-constant-condition
+      trainingWarning = (
+        <div className="feature">
+          Enrollment history does not include flight training, on-the-job-training, apprenticeship training, or correspondence training.
+        </div>
+      );
+    }
+
+    return (
+      <div>
+        <h3 className="section-header">Enrollment History</h3>
+        <div className="section-line">This information is current as of {today}</div>
+        <div className="usa-grid-full section-line">
```

Comment:
Something I did in this PR (https://github.com/department-of-veterans-affairs/vets-website/pull/5717) is extract out this markup into a function that takes the label and the value to be rendered, and renders this markup for each line, since it's pretty repetitive and there are many lines throughout the page that are formatted exactly like this one. Perhaps I could extract that out into a helper to be used across components? Not necessarily for this PR, can do this refactor in a future PR.